paper.md

```diff
@@ -0,0 +1,159 @@
+## General usage
+
+The LINCC-Frameworks Python Project Template automates the setup of the above processes for Python projects hosted on GitHub. The only direct dependency is copier[^7] [@copier], which is used as the engine to generate new projects from the LF-PPT with a specific directory structure, the requested configuration and stub files. To hydrate a new or existing project with the LF-PPT via copier the user calls:
+
+```
+copier copy gh:lincc-frameworks/python-project-template <new/project/directory>
+```
+
+A questionnaire is presented to configure the project and establish the various features of the template to include. After the directory structure and files have been generated, the user should run the included initialization script to configure the local git repository and install the new package in the virtual environment:
+
+```
+cd <new/project/directory>
+bash .initialize-new-project.sh
+```
+
+The process is designed so as not to require significant time, and, thus, if the user is unhappy with the generated project, they can simply delete and recreate it. Depending on the options selected, some additional configuration may be required, such as registering with ReadTheDocs or PyPI. To assist the user, a customized, post-creation checklist is generated.
+
+## Existing projects
+
+In addition to creating new projects from scratch, the LF-PPT can be applied to existing projects to incorporate the features it provides. Often a more established project will require more effort to apply the LF-PPT. However, LINCC-Frameworks has had success applying the template to multiple legacy projects. A collection of tips for applying the template to existing projects is included in the documentation http://lincc-ppt.readthedocs.io/.
+
+## Updating projects
+
+As best practices evolve and new tools are introduced, the LF-PPT will incorporate those into the template. Updates to the template can be applied to existing projects with minimal effort, allowing the users to focus on science and not software maintenance.
+
+```
+copier update
+```
+
+## Hibernating projects
+
+Scientific projects often go into periods of hibernation when not under active development or use, and are often challenging to revive. With the LF-PPT, hibernating projects should be much easier to reactivate. Automatically scheduled smoke tests and dependabot[^8] integration ensure that the code and dependencies continue to function correctly without significant interaction from the maintainers.
+
+## Recent applications
+
+The LF-PPT has been applied to multiple LINCC Frameworks projects including LSDB[^9], Hipscat[^10], and TAPE[^11], short term collaborations such as Sorcha[^12], SuperPhot+[^13], DeepDISC[^14], and MacCauff[^15], and external project including kcorrect[^16] and FlexCode[^17] Additionally a project specific version was forked from the project and has been applied to all RAIL packages[^18].
+
+# Acknowledgments
+
+This project is supported by Schmidt Sciences.
```

Comment:
I was curious whether there are any community members we might want to acknowledge - e.g. people who've submitted helpful feedback and feature requests, or who helped us advertise the work within LSST SCs, etc.?

Reply:
Let me think on this one a little bit, I'm sure there's a reasonable list that I can add here.

Reply:
Yes, that's a good idea as well.